website/client/src/components/admin-panel/index.vue

```diff
@@ -0,0 +1,754 @@
+                  @click="saveHero()"
+                >
+                  Save
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  .accordion-group .accordion-group {
+    margin-left: 1em;
+  }
+  h3 {
+    margin-top: 2em;
+  }
+  h4 {
+    margin-top: 1em;
+  }
+  .expand-toggle::after {
+    margin-left: 5px;
+  }
+  .subsection-start {
+    margin-top: 1em;
+  }
+  .form-inline {
+    margin-bottom: 1em;
+    input, span {
+      margin-left: 5px;
+    }
+  }
+  .ownedItem {
+    font-weight: bold;
+  }
+</style>
+
+<script>
+import moment from 'moment';
+
+import markdownDirective from '@/directives/markdown';
+import styleHelper from '@/mixins/styleHelper';
+import { mapState } from '@/libs/store';
+import content from '@/../../common/script/content';
+import notifications from '@/mixins/notifications';
+
+export default {
+  components: {
+  },
+  directives: {
+    markdown: markdownDirective,
+  },
+  mixins: [notifications, styleHelper],
+  data () {
+    return {
+      hero: {},
+      heroID: '',
+      content,
+      collatedItemData: {},
+      expandPriv: false,
+      expandAuth: false,
+      expandParty: false,
+      expandAvatar: false,
+      expandItems: false,
+      expandItemType: {
+        eggs: false,
+        hatchingPotions: false,
+        food: false,
+        pets: false,
+        mounts: false,
+        quests: false,
+        gear: false,
+        special: false,
+      },
+      expandUpdateItems: false,
+      expandContrib: false,
+      itemTypes: ['eggs', 'hatchingPotions', 'food', 'pets', 'mounts', 'quests', 'gear', 'special'],
+    };
+  },
+  computed: {
+    ...mapState({ user: 'user.data' }),
+    questStatus () {
+      if (!this.hero.party || !this.hero.party.quest) return 'No';
+      const questKey = this.hero.party.quest.key || '';
+      if (this.hero.party.quest.RSVPNeeded) {
+        if (questKey) return `${questKey} : Invitation waiting.`;
+        return 'BUG! '
+          + 'Invitation is waiting but Quest "key" is not assigned. '
+          + 'Party might or might not have a Quest. '
+          + 'An admin needs to delete the invitation or assign the Quest key.';
+      }
+      if (questKey) {
+        return `${questKey} : Invitation has been accepted. `
+          + 'Quest is either running or still in invitation stage.';
+      }
+      return 'No (party might have a quest but user is not in it).';
+    },
+  },
+  // async mounted () {
+  // },
+  methods: {
+    authMethodExists (authMethod) {
+      if (this.hero.auth[authMethod] && this.hero.auth[authMethod].length !== 0) return true;
+      return false;
+    },
+    formatDate (inputDate) {
+      const date = moment(inputDate).utcOffset(0).format('YYYY-MM-DD HH:mm');
+      return `${date} UTC`;
+    },
+    formatTimeZone (inputTimeZoneOffset) {
```

Comment:
filter

Reply:
also done in d57008704e7ebf9ef8c27ed461a6423397302794